src/panels/config/integrations/integration-panels/dhcp/dhcp-config-panel.ts

```diff
@@ -0,0 +1,111 @@
+import type { CSSResultGroup, TemplateResult, PropertyValues } from "lit";
+import { html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import type { UnsubscribeFunc } from "home-assistant-js-websocket";
+import type { LocalizeFunc } from "../../../../../common/translations/localize";
+import type { DataTableColumnContainer } from "../../../../../components/data-table/ha-data-table";
+import { extractSearchParamsObject } from "../../../../../common/url/search-params";
+import "../../../../../components/ha-fab";
+import "../../../../../components/ha-icon-button";
+import "../../../../../layouts/hass-tabs-subpage-data-table";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import type { DHCPDiscoveryData } from "../../../../../data/dhcp";
+import { SubscribeMixin } from "../../../../../mixins/subscribe-mixin";
+
+import { subscribeDHCPDiscovery } from "../../../../../data/dhcp";
+
+@customElement("dhcp-config-panel")
+export class DHCPConfigPanel extends SubscribeMixin(LitElement) {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public route!: Route;
+
+  @property({ attribute: false }) public mac_address?: string;
```

Comment:
If this is just used in the component, you can make it a state instead of a property to define that it is part of the component state.

```suggestion
  @state() public mac_address?: string;
```

Reply:
applied manually in b3bf63f2c